app/assets/stylesheets/templates/_bulk-action-form.scss

```diff
@@ -0,0 +1,17 @@
+.bulk-action-form {
+  .hr {
+    background-color: $color-grey-medium-light;
+  }
+
+  .form-group {
+    margin-bottom: 0;
+    .textarea {
+      margin-bottom: 0;
+    }
+  }
+  .form-warnings {
+    p:last-child {
+      margin-bottom: $s35;
+    }
+  }
+}
```

Comment:
The adjustments to margin-bottom were added in order to allow for the "character limit" help text to be positioned directly below the text areas